src/connector/src/source/cdc/external/mod.rs

```diff
@@ -215,8 +217,11 @@ pub struct ExternalTableConfig {
 impl ExternalTableConfig {
     pub fn try_from_btreemap(
```

Comment:
To be consistent with other implementation, here the `fill_secret` should be done outside.

Reply:
They reason `ExternalTableConfig` is different from others is it doesn't use `ConnectorProperties` somehow. @StrikeW